sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/json/JSONOptions.scala

```diff
@@ -76,6 +76,9 @@ private[sql] class JSONOptions(
   // Whether to ignore column of all null values or empty array/struct during schema inference
   val dropFieldIfAllNull = parameters.get("dropFieldIfAllNull").map(_.toBoolean).getOrElse(false)
 
+  // Whether to ignore column of all null during json generating
+  val structIngoreNull = parameters.getOrElse("structIngoreNull", "true").toBoolean
```

Comment:
is it specific to struct type column? if not how about naming it `ignroeNullColumns`?

Reply:
It works on StructType, including struct field and struct inner data

Reply:
how about top-level columns?

Reply:
Yeap, it also works on that

Reply:
then shall we pick a better name here?

Reply:
okey, ignoreNullFields is much better than structIgnoreNull, I'll change it.